aiohttp_jinja2/helpers.py

```diff
@@ -0,0 +1,74 @@
+"""
+useful context filters, see
+http://jinja.pocoo.org/docs/dev/api/#jinja2.contextfilter
+"""
+import jinja2
+
+
+@jinja2.contextfilter
+def reverse_url(context, name_, **parts):
```

Comment:
I'd rather leave `name_` as `__aiohttp_jinja2_route_name` to prevent names conflict here.